MapboxSearch/sdk/src/androidTest/assets/sbs_responses/suggestions-error-response.json

```diff
@@ -0,0 +1 @@
+{"message":"Wrong arguments"}
```

Comment:
According to SBS spec, it is possible that we receive error body with the following structure:
```json
{
    "status_code":400,
    "error":"some error message",
    "version":"some version",
    "code":"some code"
}
```

Do we have tests for such messages in our codebase?

P.S: To receive error in the previously mentioned format, you can make SBS category search w/o `origin ` and `proximity` parameters.

Reply:
Good note, thanks. I've added a test case for this structure, both cases work.